requirements-test.txt

```diff
@@ -1,4 +1,5 @@
 openwisp-utils[qa,selenium] @ https://github.com/openwisp/openwisp-utils/archive/refs/heads/1.3.tar.gz
+openwisp-notifications @ https://github.com/openwisp/openwisp-notifications/archive/refs/heads/1.3.tar.gz
```

Comment:
Thanks for the contribution! However, we cannot accept changes to dependencies or core settings at this time as they complicate our CI/CD pipeline. Please revert those specific changes so we can focus on reviewing the core logic. Since we’re currently handling a high volume of PRs, we have to prioritize "clean" submissions that don't introduce build risks. If this task feels a bit complex, I’d be happy to point you toward some "Good First Issues" that are easier to get through the pipeline!